packages/frontend/src/ScreenController.tsx

```diff
@@ -121,6 +121,9 @@ export default class ScreenController extends Component {
       await this.unSelectAccount()
       this.selectedAccountId = accountId
       ;(window.__selectedAccountId as number) = accountId
+      // forcing to load settings here so when we for example switch to Settings
```

Comment:
Shouldn't we remove [`SettingsStoreInstance.effect.load()` in `MainScreen.tsx`](https://github.com/deltachat/deltachat-desktop/blob/42ab00b45df1b0ebd9e95b08dd99e49e660f10f0/packages/frontend/src/components/screens/MainScreen.tsx#L133-L136) then?

And shouldn't we `await` it, if this is the way we want to go?

Reply:
I don't see the point in asyncing except for future-proofing which I don't think is a good idea, since everything there inside is `await`-ed already so we go back to ScreenController only after all the async things are done.

I could test the MainScreen change I just expected that we might not call ChangeAccount on start or there's some other case where this is covered.

Reply:
> except for future-proofing

IDK if it's just about future-proofind. Maybe there is another undiscovered bug already that is related to this.

Thought it's best to tackle it while we're at it. But fine. An improvement is an improvement.